linker/shared/src/main/scala/org/scalajs/linker/backend/emitter/InternalOptions.scala

```diff
@@ -10,19 +10,24 @@
 package org.scalajs.linker.backend.emitter
 
 private[emitter] class InternalOptions private (
-    val optimizeBracketSelects: Boolean) {
+    val optimizeBracketSelects: Boolean
+) {
 
   def withOptimizeBracketSelects(optimizeBracketSelects: Boolean): InternalOptions =
     copy(optimizeBracketSelects = optimizeBracketSelects)
 
   private def copy(
-      optimizeBracketSelects: Boolean = this.optimizeBracketSelects): InternalOptions = {
+      optimizeBracketSelects: Boolean = this.optimizeBracketSelects
+  ): InternalOptions = {
     new InternalOptions(
         optimizeBracketSelects)
   }
 }
 
 private[emitter] object InternalOptions {
+  val UseBigIntForLongs =
+    System.getProperty("org.scalajs.linker.backend.emitter.usebigintforlongs", "false") == "true"
```

Comment:
:((( very sad face.

Could we make a `scalajs` private `OutputMode` instead?

By the way I think this is an excellent test case for the linker API design: If we need to resort to system properties (= global static variables) for this kind of thing, we maybe need to think some more.